README.md

```diff
@@ -59,11 +59,37 @@ should be automated/scripted as part of the release process.
    There is also `docs-staging-v2` (production) and `docs-staging-v2-dev`
    (development). Use `-staging` until your HTML format is confirmed to be
    correct.
-1. That's it! doc-pipeline periodically runs, generates the HTML for new
+1. That's it! `doc-pipeline` periodically runs, generates the HTML for new
    `docfx-*` tarballs, and uploads the resulting HTML to the same bucket. The
    HTML has the same name as the DocFX tarball, except it doesn't have the
    `docfx` prefix.
 
+### Cross references
+
+DocFX supports [cross references using xrefmap files](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
+Each file maps a UID to the URL for that object. The xref map files are
+automatically generated when DocFX generates docs. One generation job can refer
+to other xref map files to be able to link to those objects.
+
+Here's how it works in `doc-pipeline`:
+
+1. When we convert the YAML to HTML, we upload two things:
+   1. The resulting HTML content (in a tarball).
+   1. The xref map file to the `xrefs` directory of the bucket. You can see them
+      all using `gsutil ls gs://docs-staging-v2/xrefs`.
+1. If one package wants to use the xref map from another package, you need to
+   configure it.
+   1. When you generate the `docs.metadata` using `docuploader`, use the `xrefs`
+      argument to specify the xref map files you need. Use the following format:
+      * `devsite://library[@version]`: use the xref from `library`. If no version
```

Comment:
This should include [lang](https://github.com/googleapis/doc-pipeline/blob/master/docpipeline/generate.py#L279) too like: devsite://lang/library@version

Reply:
Might help to include an example too like devsite://java/gax will grab the latest while devsite://java/gax@1.63.3 will grab that version

Reply:
Done.